doc/dev/index.rst

```diff
@@ -841,6 +865,11 @@ An individual test from the ceph-disk suite can be run by adding the
       --suite ceph-disk/basic \
       --filter 'ceph-disk/basic/{distros/ubuntu_14.04.yaml tasks/ceph-disk.yaml}'
 
+.. note: To run a standalone test, ``--suite`` alone is sufficient. If you
```

Comment:
s/standalone/specific/ s/standalone/given/ ? I my mind standalone is roughly the same as a test defined by a single yaml file, which is possible, although uncommon.